src/dotnet/Chat.Service/Chats.Backend.cs

```diff
@@ -329,6 +332,12 @@ public virtual async Task<string[]> GetOwnedChatIds(string userId, CancellationT
         entry = dbEntry.ToModel();
         context.Operation().Items.Set(entry);
         context.Operation().Items.Set(isNew);
+
+        if (!entry.Content.IsNullOrEmpty() && !entry.IsStreaming && entry.Type == ChatEntryType.Text) {
```

Comment:
Probably the final API will be like: `context.Operation().AddEvent(...)`.

Reply:
Yep